### PR TITLE
Switch Polish to Weblate

### PR DESCRIPTION
**Pull request by rtsisyk**
This patch overwrites Polish Markdowns with translations from Weblate. Any further edits should be done through Weblate.

Signed-off-by: Roman Tsisyk <roman@tsisyk.com>

### CODE REVIEW

**Comment by lunarna-gh** [comment]
It's alright, will probably be easier for me as well

File: content/_index.pl.md [hunk 1]
```diff
@@ -1,121 +1,127 @@
...
+**Organic Maps** to jedna z niewielu obecnie aplikacji, która obsługuje 100% funkcji bez aktywnego połączenia z Internetem. Zainstaluj Organic Maps, pobierz mapy, wyrzuć kartę SIM (przy okazji operator stale Cię śledzi) i wybierz się na tygodniową wycieczkę na jednym ładowaniu baterii bez żadnego bajtu wysłanego do sieci.
 
-### Pobierz i zainstaluj Mapy Organiczne z [AppStore][appstore], [Google Play][googleplay], [FDroid][fdroid], [Huawei AppGallery][appgallery] {#install}
+### Pobierz i zainstaluj Organic Maps z [AppStore][appstore], [Google Play][googleplay], [FDroid][fdroid], [Huawei AppGallery][appgallery] {#install}
 
 {{ badges(lang='pl') }}
 
-{{ screenshot(src='/images/screenshots/hiking.jpg', alt='Wspinaczka') }}
+{{ screenshot(src='/images/screenshots/hiking.jpg', alt='Turystyka piesza')
+}}
```
Comment (biodranik) [inline content/_index.pl.md:19]: @rtsisyk Why is it on the next line?

File: content/_index.pl.md [hunk 2]
```diff
@@ -1,121 +1,127 @@
...
+
 {{ screenshot(src='/images/screenshots/prague.jpg', alt='Praga') }}
-{{ screenshot(src='/images/screenshots/search.jpg', alt='Wyszukiwanie offline') }}
-{{ screenshot(src='/images/screenshots/dark.jpg', alt='Nawigacja w motywie ciemnym') }}
 
-## Funkcje
+{{ screenshot(src='/images/screenshots/search.jpg', alt='Wyszukiwanie
+offline') }}
```
Comment (biodranik) [inline content/_index.pl.md:24]: @rtsisyk ditto

File: content/_index.pl.md [hunk 3]
```diff
@@ -1,121 +1,127 @@
...
 
-Organic Maps to doskonała aplikacja towarzysząca dla podróżników, turystów, wędrowców i rowerzystów:
+{{ screenshot(src='/images/screenshots/dark.jpg', alt='Nawigacja w trybie
+ciemnym') }}
```
Comment (biodranik) [inline content/_index.pl.md:27]: @rtsisyk ditto

File: content/_index.pl.md [hunk 4]
```diff
@@ -1,121 +1,127 @@
...
+Odrzuć inwigilację - ogarnij swoją wolność.
 
-<a href="#install"><strong>Wypróbuj Organic Maps!</strong></a>
+**[Daj szansę Organic Maps!](#install).**
 
 ## Kto płaci za darmową aplikację?
 
-Aplikacja jest darmowa dla wszystkich. Proszę [przekaż darowiznę](@/donate/index.pl.md) by nas wspierać!
+Aplikacja jest darmowa dla wszystkich. Prosimy o
+[podarunek](@/donate/index.md), aby nas wesprzeć!
```
Comment (biodranik) [inline content/_index.pl.md:91]: @rtsisyk should it be index.pl.md?

File: content/donate/index.pl.md [hunk 5]
```diff
@@ -1,76 +1,84 @@
...
 
 {{ crypto_table(lang='pl') }}
 
-## Czy można pomóc w inny sposób?
+## Czy można nam pomóc w jakiś inny sposób?
 
-Tak! Jest wiele innych sposobów by wesprzeć Organic Maps. Proszę zobacz
-stronę [Przyczyń się](@/support-us/index.pl.md) by uzyskać więcej informacji.
+Tak! Istnieje wiele sposobów na wsparcie Organic Maps. Więcej szczegółów
+znajdziesz na stronie [Wesprzyj nas](@/support-us/index.md).
```
Comment (biodranik) [inline content/donate/index.pl.md:81]: @rtsisyk index.pl.md

File: content/support-us/index.pl.md [hunk 6]
```diff
@@ -1,39 +1,44 @@
...
-Organic Maps jest darmową aplikacją z otwartym kodem źródłowym. Jest wolna od reklam, nie zbiera Twoich danych osobowych,
-i jest rozwijana przez entuzjastów z pomocą społeczności.
+Organic Maps to darmowa aplikacja typu open-source. Jest wolna od reklam,
+nie zbiera Twoich danych osobowych i jest rozwijana przez nielicznych
+entuzjastów z pomocą społeczności.
+
+Wspierać rozwój można na różne sposoby:
 
-Jest wiele sposobów na przyczynienie się do rozwoju Organic Maps:
+- [Podaruj](@/donate/index.md)! Każdy dolar lub euro się liczy i pomaga nam
```
Comment (biodranik) [inline content/support-us/index.pl.md:16]: @rtsisyk index.pl.md

**Comment by lunarna-gh** [comment]
Huh, I just noticed a lot of the translations were changed - were they pending translations from @strebski? Some are definitely improvements but I wish I got a chance to discuss these beforehand, it's currently way too literal in places and very inconsistent in formality, plurality and terminology

**Comment by strebski** [comment]
Hi, @lunarna-gh feel free to discuss any changes I'm making, I'm open for all kind of criticism ;-) Best wishes!

File: content/_index.pl.md (hunk 4 above)
Comment (strebski) [reply content/_index.pl.md:91]: Dear @biodranik - in my opinion: yes. "Darowizna" has strong legal context in Polish language (related with taxes etc.); "podarunek" is very soft, related with financial gift, but without above problem. "Podarunek" is Russian "подарок" ;-)

Comment (biodranik) [reply content/_index.pl.md:91]: Thanks!
@rtsisyk links are not fixed yet. They lead to EN pages for many languages.

**Pull request merged**